src/main/core/worker.c

```diff
@@ -551,7 +551,7 @@ void worker_sendPacket(Host* srcHost, Packet* packet) {
      * control has problems responding to packet loss */
     if (bootstrapping || chance <= reliability || packet_getPayloadLength(packet) == 0) {
         /* the sender's packet will make it through, find latency */
-        gfloat latency = worker_getLatencyForAddresses(srcAddress, dstAddress);
+        gfloat latency = worker_getLatencyForAddresses(srcAddress, dstAddress, true);
```

Comment:
1. Is it worth adding the extra `bool updateMinTimeJump` argument to all of the `getLatency*` functions, rather than just calling one extra new function call here to explicitly update the min jump time? (I think this is the only place we pass `true`.)
2. If we want to avoid too much locking in the controller when trying to write the new jump time, each worker could keep it's own local `minJumpTime` state and only ask the controller to update when it finds a smaller local jump time.

Reply:
>     1. Is it worth adding the extra `bool updateMinTimeJump` argument to all of the `getLatency*` functions, rather than just calling one extra new function call here to explicitly update the min jump time? (I think this is the only place we pass `true`.)

My reason against doing it this way is that then we need another group of functions `worker_updateMinJumpTime()` -> `manager_updateMinJumpTime()` -> `controller_updateMinJumpTime()`, and we have to change the return type of all of our latency functions from `gdouble` to `SimulationTime`, but I added these now.

>     2. If we want to avoid too much locking in the controller when trying to write the new jump time, each worker could keep it's own local `minJumpTime` state and only ask the controller to update when it finds a smaller local jump time.

Added.